pkg/daemon/controller/controller.go

```diff
@@ -327,6 +376,19 @@ func (c *Controller) handleVxlanInterfaceNeighEvent() error {
 					return fmt.Errorf("parse vtep mac %v failed: %v",
 						node.Annotations[constants.AnnotationNodeVtepMac], err)
 				}
+			} else if daemonfeature.MultiClusterEnabled() {
```

Comment:
use switch here to make logic clear?

Reply:
Maybe it's not needed, just some simple conditions.